hyperhtml/documentation/index.html

```diff
@@ -534,26 +531,28 @@ <h3 id="essentials-6">Events Attributes</h3>
               );
               </code></pre>
               <p>
-                You can test above example directly on <a href="https://codepen.io/WebReflection/pen/Yxxqwy?editors=0010">Code Pen</a>.
+                View the example above on <a href="https://codepen.io/WebReflection/pen/Yxxqwy?editors=0010">Code Pen</a>.
               </p>
               <p>
                 Bear in mind that runtime assigned listeners will inevitably remove and re-add themselves since
-                two functions, even if with an identical look, are always different in JavaScript.
-                This means that while the previous example works fine for a demo purpose,
-                that <em>onsubmit</em> listener should be defined elsewhere instead of inline.
+                two functions, even if identical, are always different in JavaScript.
+                This means that while the previous example works fine for demo purposes,
+                the <em>onsubmit</em> listener should be defined elsewhere and not inline.
               </p>
               <h3 id="essentials-7">Partial Attributes</h3>
               <p>
-                Since template literals interpolations can also contain template literals,
-                compromising performance over-complicating attributes operations
-                has been considered <acronym title="You Ain't Gonna Need It">YAGNI</acronym>
-                during the architecture design of the parser.
+                Partial attributes are not supported,
+                and most likely will never be supported because there are
+                many ways to achieve the same goal via a single attribute.
```

Comment:
"via a single attribute" ... definition?

It's not about single attribute, it's about defining even multiple attribute content as single interpolation.